### PR TITLE
xenos can now fully evo pre-shutters

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Ties evolution to quick build.
If you have access to quick build, you can evo as much as you want.
## Why It's Good For The Game
Evo restrictions exist as a CM/Survivor era relic where there was an expectation of fighting during the prep phase.
Now with this gone, it no longer makes sense for xeno evolution to be restricted in prep phase.
## Changelog
:cl:
balance: Xenos no longer need evo points to evolve in prep phase
/:cl:

### CODE REVIEW

**Comment by Barnet2** [comment]
RIP survivor events but yeah, if T3 slots aren't fully filled when you latejoin as xeno this will be nice so you don't get killed out of it

**Review by Kuro020 — APPROVED** [review]

**Comment by yyzsong** [comment]
surv mains owned

**Comment by Wisemonster** [comment]
This would be nice, considering now the xenos can go widow/hivelord round-start to maze, then caste swap to whatever caste they actually wanna play when shutters are gonna drop.

**Comment by TiviPlus** [comment]
I just know people are gonna use it to hog rav instantly lmao

File: code/modules/mob/living/carbon/xenomorph/evo_datum.dm [hunk 1]
```diff
@@ -57,15 +57,23 @@
 
 /// Some data to update the UI with the current evolution status
 /datum/evolution_panel/ui_data(mob/living/carbon/xenomorph/xeno)
-	. = list()
+	var/list/data = list()
+
+	data["bypass_evolution_checks"] = SSresinshaping.active
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/evo_datum.dm:62]: Why isn't this just in can evolve

Comment (Xander3359) [reply code/modules/mob/living/carbon/xenomorph/evo_datum.dm:62]: initially I had it there and it didn't work well with the UI.
I kinda want to make it where if this is active there is some UI indicator but I keep fucking up the UI whenever I try to make changes to it.

**Pull request merged**